astroquery/ibe/core.py

```diff
@@ -107,7 +118,8 @@ def query_region(
         # Raise exception, if request failed
         response.raise_for_status()
 
-        return Table.read(response.text, format='ipac', guess=False)
+        return commons.parse_votable(
+            encode(response.text)).get_first_table().to_table()
```

Comment:
`response.content` is the encoded version; you don't need the python version hacks above

Reply:
Thanks.